matminer/featurizers/structure/bonding.py

```diff
@@ -805,8 +805,8 @@ def get_equiv_sites(self, s, site):
         sga = SpacegroupAnalyzer(s, symprec=0.01)
         sg = sga.get_space_group_operations
         sym_data = sga.get_symmetry_dataset()
-        equiv_atoms = sym_data["equivalent_atoms"]
-        wyckoffs = sym_data["wyckoffs"]
+        equiv_atoms = sym_data.equivalent_atoms
+        wyckoffs = sym_data.wyckoffs
```

Comment:
Do we need to break old spglib compatibility here? The deps of matminer are really not kept super up to date anymore, so I'm worried that adding new lower pins for stuff like this will cause breakages down the line. How about something like:

```suggestion
        equiv_atoms = getattr(sym_data, "equivalent_atoms", sym_data["equivalent_atoms"])
        wyckoffs = getattr(sym_data, "wyckoffs", sym_data["wyckoffs"])
```

Reply:
Good with me, reverted the lower pin on spglib and implemented your suggestion!

Reply:
Thanks! Let's see if the test pass with the current pin (2.4), and if they do, we can bump the pinned version to 2.5.